packages/vitest/src/node/cli.ts

```diff
@@ -11,6 +11,18 @@ const CLOSE_TIMEOUT = 1_000
 
 const cli = cac('vitest')
 
+type DevCLIConfig = ApiConfig & UserConfig
+interface DevCLIOptions extends DevCLIConfig {}
+
+function buildDevOptions<Options extends DevCLIOptions>(
```

Comment:
do we really need this function now? doesn't seem to do anything useful

Reply:
there is one small problem: if we run `vitest -o --api --api.xxx yyy` then will run with defaults...

Reply:
why would we run `--api --api.xxx` tho?